### ISSUE TITLE
Password recovery edge

### ISSUE THREAD

**Issue by Error336070** [quasar/Quasar#1143]
### Quasar version

1.4.1

### Server installed .NET version

.NET 6

### Server operating system

Windows 10/Server 2019/2016

### Client installed .NET version

.NET 6

### Client operating system

Windows 10/Server 2019/2016

### Build configuration

Release

### Describe the bug

I couldnt find any places to ask so im quite sorry for asking here, how can i use the password recovery tool?
![image](https://user-images.githubusercontent.com/71385222/227935400-8c21bf0a-e9be-4562-8fb6-43be4efab902.png)
i have this computer with the rat on and i have absolutely 0 idea on how to do a password recovery. the computer Uses Microsoft edge as a browser and i get no results when i use quasar rat simply because I'm quite dumb and have no idea how to use this. i have tried another rat that did this automatically which worked but it has been failing to work so far and quasar overall is a better option so i'd love to get some help on this.

### How to reproduce

opening password recovery tool in the program

### Expected behavior

got no idea

### Actual behavior

basically nothing happened bc i dont know how to use quasar rat's password recovery tool
